backend/plugins/mongolian_api/src/modules/erkhet/utils/consumeInventory.ts

```
@@ -73,50 +73,50 @@ export const consumeInventory = async (subdomain, doc, old_code, action) => {
       subUoms.unshift({
         uom: doc.sub_measure_unit_code,
-        ratio: doc.ratio_measure_unit
+        ratio: doc.ratio_measure_unit,
       });
 
       document.subUoms = subUoms;
     }
 
     if (config.consumeDescription) {
-      doc.description = eval("`" + config.consumeDescription + "`");
+      doc.description = eval('`' + config.consumeDescription + '`');
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)